Makefile

```diff
@@ -34,9 +38,18 @@ CFLAGS += -fsigned-char $(DEVLIBS) \
 	-I/usr/include/SDL \
 	-I$(SDKSTAGE)/opt/vc/include -I$(SDKSTAGE)/opt/vc/include/interface/vcos/pthreads \
 	-I$(SDKSTAGE)/opt/vc/include/interface/vmcs_host/linux \
-	-I/usr/include/glib-2.0 -I/usr/lib/arm-linux-gnueabihf/glib-2.0/include \
-	-O3 -ffast-math -fno-builtin -fsingle-precision-constant \
-	-Wall -Wno-sign-compare -Wunused -Wpointer-arith -Wcast-align -Waggregate-return -Wshadow 
+	-I/usr/include/glib-2.0 -I/usr/lib/arm-linux-gnueabihf/glib-2.0/include
+
+# Platform specific
+ifeq ($(PLATFORM),rpi2)
+CFLAGS += -mcpu=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard
+else
+CFLAGS += -march=armv6j -mfpu=vfp -mfloat-abi=hard
+endif
+
+CFLAGS += -O3 -ffast-math -fno-builtin -fsingle-precision-constant \
+	-fno-common -mstructure-size-boundary=32 -fweb -frename-registers \
+	-Wall -Wno-sign-compare -Wunused -Wpointer-arith -Wcast-align -Waggregate-return -Wshadow
 
```

Comment:
I see.  Yep I put back what I saw wasn't obviously covered by O3 in the documentation.  I have been using what was compiled on my RPi2, with those set, and haven't noticed any significant differences.  My attempts to recompile for the PS3 wireless fix ballooned into delving into the gcc Make process.  Apologies for my misunderstanding of the previous changes.